build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'java'
```

Comment:
Did we want to explicitly set the Java version to 1.8 below the list of plugins as was done in my initial pull request?
```
sourceCompatibility = 1.8
targetCompatibility = 1.8
```
I understand it's currently set to 1.7 at https://github.com/jusu/Elephant/blob/master/build.xml#L9, but I see no problem taking this opportunity to move to 1.8 in both `build.gradle` and `build.xml`. The last publicly available update of Java 7 was [2015-04-14](https://en.wikipedia.org/wiki/Java_version_history#Java_7_updates), so most people should be running Java 8.

Reply:
Done